src/gui/tray/activitylistmodel.cpp

```diff
@@ -818,5 +827,25 @@ void ActivityListModel::slotRemoveAccount()
     _showMoreActivitiesAvailableEntry = false;
 }
 
+void ActivityListModel::replyMessageSent(const int activityIndex, const QString &message)
+{
+    if (activityIndex < 0 || activityIndex >= _finalList.size()) {
+        qCWarning(lcActivity) << "Couldn't trigger action on activity at index" << activityIndex << "/ final list size:" << _finalList.size();
+        return;
+    }
+
+    _finalList[activityIndex]._talkNotificationData.messageSent = message;
+
+    emit messageSent();
 }
 
+QString ActivityListModel::talkReplyMessageSent(const int activityIndex) const
```

Comment:
would it not be better to name them
`replyMessageSent` -> `setReplyMessageSent`
`talkReplyMessageSent` -> `replyMessageSent`

because if I understand they have a getter/setter logic